### PR TITLE
Stateless agent

### PR DESCRIPTION
**Pull request by dapatil211**
Note, docstrings still need to be updated. Please review other things in the meantime.

### CODE REVIEW

File: hive/agents/dqn.py [hunk 1]
```diff
@@ -264,13 +263,13 @@ def act(self, observation):
...
-            and self._state["episode_start"]
+            and state is None
         ):
             self._logger.log_scalar("train_qval", torch.max(qvals), self._timescale)
-            self._state["episode_start"] = False
-        return action
+            state = {}
+        return action, state
 
-    def update(self, update_info):
```
Comment (hnekoeiq) [inline hive/agents/dqn.py:273]: IMO, agent_state would be less confusing.

File: hive/runners/single_agent_loop.py [hunk 2]
```diff
@@ -85,7 +85,9 @@ def __init__(
         )
         self._stack_size = stack_size
 
-    def run_one_step(self, environment, observation, episode_metrics, transition_info):
+    def run_one_step(
```
Comment (hnekoeiq) [inline hive/runners/single_agent_loop.py:89]: This looks weird. Do you know why Black does that?

**Review by hnekoeiq — APPROVED** [review]
Two minor comments. Other than that, it looks good.

File: hive/runners/single_agent_loop.py (hunk 2 above)
Comment (dapatil211) [reply hive/runners/single_agent_loop.py:89]: I think it's because the line is longer now, and so it'd go above the 88 column threshold

**Comment by mrsamsami** [comment]
Except for the naming, which Hadi also mentioned, everything looks good. I'd recommend changing the `state` variable name to `memory` because it basically stores information about the past and present (not only for DRQN).

**Review by mrsamsami — APPROVED** [review]
LGTM.

**Pull request merged**